diff --git a/lib/common/widgets/scaffold/mini_scaffold.dart b/lib/common/widgets/scaffold/mini_scaffold.dart
index d1a0ca667..2b7760943 100644
--- a/lib/common/widgets/scaffold/mini_scaffold.dart
+++ b/lib/common/widgets/scaffold/mini_scaffold.dart
@@ -72,6 +72,7 @@ class MiniScaffoldState extends State<MiniScaffold>
     }
 
     final LocalHistoryEntry entry = LocalHistoryEntry(
+      popGestureEnabled: true,
       onRemove: () {
         if (!removedEntry &&
             _currentBottomSheet?.widget == bottomSheet &&
